templates/login.html

```diff
@@ -0,0 +1,36 @@
+{{ define "content" }}
+
+<h2 class="auth-main__title">
+  Login to your account
+</h2>
+
+{{ if .notices }}
+  <div class="alert alert--success">
+    {{ range $notice := .notices }}
+      {{ $notice }}
+    {{ end }}
+  </div>
+{{ end }}
+
+{{ if .errors }}
+  <div class="alert alert--danger">
+    {{ .errors }}
+  </div>
+{{ end }}
+
+
+<form class="mt-8 space-y-6" action="/login" method="POST">
+  <input name="email" type="email" required placeholder="Email address" value={{.email}}>
+  <input name="password" type="password" required placeholder="Password">
+
+  <button type="submit">
+    Login
+  </button>
+</form>
+
+<p class="auth-main__content--small">
+  Don't have account yet?
```

Comment:
English:

```suggestion
  Don't have an account yet?
```

Reply:
Fixed in b47ee9e71f5806d2e006533e2e3dfda9f34c5066